### PR TITLE
[MNG-7606] add IT

### PR DESCRIPTION
**Pull request by kwin**
https://issues.apache.org/jira/browse/MNG-7606

### CODE REVIEW

**Review by cstamas — APPROVED** [review]

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        verifier.deleteDirectory( "target" );
+        verifier.deleteArtifacts( "org.apache.maven.its.mng7606" );
+        try
+        {
+            verifier.executeGoal( "validate" );
+            verifier.verifyErrorFreeLog();
+        }
+        finally
+        {
+            verifier.resetStreams();
```
Comment (slawekjaranowski) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java:47]: Deprecated - not needed - please also look for other deprecated methods of Verifier.

**Comment by cstamas** [comment]
Hm, and issue is wrong: https://issues.apache.org/jira/browse/MNG-7606

**Comment by kwin** [comment]
> Hm, and issue is wrong: https://issues.apache.org/jira/browse/MNG-7606

Fixed commit message in https://github.com/apache/maven-integration-testing/commit/37a5d646c879f8b46d47c98966041aa3e85ae8bc

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java (hunk 1 above)
Comment (kwin) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java]: Fixed in https://github.com/apache/maven-integration-testing/commit/37a5d646c879f8b46d47c98966041aa3e85ae8bc

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * It checks that "import" scope for dependencies work
+ *
+ */
+class MavenITmng7606DependencyImportScopeTest
+    extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng7606DependencyImportScopeTest()
+    {
+        super( "[3.9.0,)" );
```
Comment (kwin) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java:20]: What would be an appropriate lower bound here?

Comment (cstamas) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java:20]: This should work with all Mavens, no?

Comment (kwin) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java]: Agreed, adjusted accordingly in https://github.com/apache/maven-integration-testing/pull/214/commits/e192fd6384aaeb266ce14b2a95a24947887dd3d9

**Review by cstamas — APPROVED** [review]

**Pull request merged**